docs/changes/coordinates/15205.api.rst

```diff
@@ -0,0 +1,3 @@
+Removed deprecated ``dms_to_degrees``, ``hms_to_hours``, ``hms_to_degrees``, ``hms_to_radians``,
+and ``hms_to_dms`` in ``astropy.coordinates.angle_formats``. Initializing ``Angle`` and ``Longitude``
+with a tuple is no longer supported and will now throw ``NotImplementedError``.
```

Comment:
As far as I can tell `astropy.coordinates.angle_formats` is supposed to be a private module, so I'm not sure we should mention it explicitly in the change log.
```suggestion
The deprecated functionality of initializing ``Angle`` or ``Longitude`` from a ``tuple`` is no longer supported.
```

Reply:
Accepted. Thanks!